debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/BinlogReader.java

```diff
@@ -101,6 +105,9 @@ public BinlogReader(String name, MySqlTaskContext context) {
         client.registerLifecycleListener(new ReaderThreadLifecycleListener());
         if (logger.isDebugEnabled()) client.registerEventListener(this::logEvent);
 
+        String gtidDmlSourceIncludes = context.config().getString(MySqlConnectorConfig.GTID_DML_SOURCE_INCLUDES);
+        gtidDmlSourceFilter = gtidDmlSourceIncludes != null ? Predicates.includes(gtidDmlSourceIncludes) : null;
```

Comment:
The `MySqlTaskContext` has a `gtidSourceFilter()` method that returns a `Predicate<String>` that you could use. If we want a config option that specifies whether the events should be filtered with the GTID sources, then this could check that and if true just use the result of `gtidSourceFilter()`.